x/pocketcore/tx.go

```diff
@@ -41,8 +39,3 @@ func ProofTx(cliCtx util.CLIContext, txBuilder auth.TxBuilder, branches [2]types
 	}
 	return util.CompleteAndBroadcastTxCLI(txBuilder, cliCtx, msg)
 }
-
-// "GenerateAAT" - Exported call to generate an application authentication token
-func GenerateAAT(keybase keys.Keybase, appPubKey, cliPubKey, passphrase string) (types.AAT, error) {
```

Comment:
AAT generation , moved or removed?

Reply:
We don't use the tx.go version of GenerateAAT, the CLI calls the keeper instead of this one.